### ISSUE TITLE
Accessibility: Limited screen reader support

### ISSUE THREAD

**Issue by sjaeger24** [donhinkelman/moodle-block_sharing_cart#284]
When using a screen reader, the identification of the basic functions “Copy elements to Sharing Cart” and “Add elements to course” is severely limited. 

The course sections are missing the alt text for the basket icon next to the section headings and the activities contained therein. The “Copy item” function is not mentioned. 

The function of the arrow icon, which inserts elements from the Sharing Cart into a section, also remains unclear and misleading. The screen reader reads that there is a list with a number of clickable elements. The number of elements refers to the elements within the section into which elements from the Sharing Cart are to be copied.

In the block, the list of items is not announced as such. The icons in front of the item names do not have alt text. Folders and (the activities they contain) cannot be distinguished from one another. The screen reader tells only the name of the closed folder or activity and that it is clickable.

The functions of the icons next to the items in the clipboard are not named, there is no alt text ("Delete item", "Copy item").
